giskard_vision/landmark_detection/detectors/metadata_detector.py

```diff
@@ -0,0 +1,25 @@
+from giskard_vision.core.detectors.metadata_scan_detector import MetaDataScanDetector
+from giskard_vision.core.detectors.metrics import MetricBase
+from giskard_vision.landmark_detection.detectors.surrogate_functions import (
+    relative_volume_convex_hull,
+)
+from giskard_vision.landmark_detection.tests.performance import NMEMean
+from giskard_vision.landmark_detection.types import PredictionResult
+
+from ...core.detectors.decorator import maybe_detector
+
+
+class NMEMeanMetric(MetricBase):
+    type_task = "regression"
+    name = "NMEMean"
+
+    def get(self, pred, truth):
+        return NMEMean.get(PredictionResult(prediction=pred[None, :]), truth[None, :])
```

Comment:
here you're breaking the initial API. 

There's already an NME metric, my point was to rather adapt the Detector API to take it as opposed to other way around.